Src/Common/RequestResponseHeaders.cs

```diff
@@ -8,12 +8,12 @@ internal static class RequestResponseHeaders
         /// <summary>
         /// Source instrumentation header that is added by an application while making http requests and retrieved by the other application when processing incoming requests.
         /// </summary>
-        public const string SourceInstrumentationKeyHeader = "x-ms-request-source-ikey";
```

Comment:
Per @lmolkova the only headers affected by her work are Parent and Root Id headers. The source and target headers have no story. She suggests no change is required here.